.github/workflows/checks.yml

```diff
@@ -1,9 +1,15 @@
 name: Run checks
 
-on: [push]
-
-permissions:
-  contents: read
+on:
+  push:
+    permissions:
+      contents: read
+  pull_request_target:
+    types:
+      - opened
+    permissions:
```

Comment:
Could we move these permissions down to the `build` job, similar to what's in [this example](https://github.com/marketplace/actions/jest-coverage-report#pull-request-number)?

Reply:
I'll try that out and see if it works.